docs/api.md

```diff
@@ -0,0 +1,392 @@
+            "protected_area": "0.000259953624046374",
+            "percent_protected": "30.4005429385137"
+        }
+    ],
+    "habitat_change_statistics": [
+        {
+            "name": "mangroves",
+            "total_area": {
+                "total_value_1996": "7540.75998764203",
+                "total_value_2007": "7281.55131473272",
+                "total_value_2008": "7304.8783090984",
+                "total_value_2009": "7296.02930815457",
+                "total_value_2010": "0.0",
+                "total_value_2015": "7235.42899839272",
+                "total_value_2016": "7240.539621204",
+                "baseline_year": 2010
+            }
+        }
+    ],
+    "species_status": {
+        "coralreefs": [
+            ["CR", 0],
+            ["EN", 7],
+            ["VU", 79],
+            ["NT", 119],
+            ["LC", 183],
+            ["DD", 22],
+            ["NE", 0],
+            ["total", 410]
+        ],
+        "mangroves": [
+            ["CR", 0],
+            ["EN", 0],
+            ["VU", 0],
+            ["NT", 0],
+            ["LC", 10],
+            ["DD", 0],
+            ["NE", 0],
+            ["total", 10]
+        ],
+        "saltmarshes": [
+            ["CR", 0],
+            ["EN", 0],
+            ["VU", 0],
+            ["NT", 0],
+            ["LC", 2],
+            ["DD", 0],
+            ["NE", 0],
+            ["total", 2]
+        ],
+        "coldcorals": [
+            ["CR", 0],
+            ["EN", 0],
+            ["VU", 0],
+            ["NT", 0],
+            ["LC", 3],
+            ["DD", 0],
+            ["NE", 0],
+            ["total", 3]
+        ],
+        "seagrasses": [
+            ["CR", 0],
+            ["EN", 0],
+            ["VU", 1],
+            ["NT", 0],
+            ["LC", 12],
+            ["DD", 1],
+            ["NE", 0],
+            ["total", 14]
+        ]
+    },
+    "countries": [
+        {
+            "name": "Comoros",
+            "iso3": "COM"
+        },
+        {
+            "name": "French Southern And Antarctic Territories",
+            "iso3": "ATF"
+        },
+        {
+            "name": "Kenya",
+            "iso3": "KEN"
+        },
+        {
+            "name": "Madagascar",
+            "iso3": "MDG"
+        },
+        {
+            "name": "Mauritius",
+            "iso3": "MUS"
+        },
+        {
+            "name": "Mayotte",
+            "iso3": "MYT"
+        },
+        {
+            "name": "Mozambique",
+            "iso3": "MOZ"
+        },
+        {
+            "name": "Reunion",
+            "iso3": "REU"
+        },
+        {
+            "name": "Seychelles",
+            "iso3": "SYC"
+        },
+        {
+            "name": "Somalia",
+            "iso3": "SOM"
+        },
+        {
+            "name": "South Africa",
+            "iso3": "ZAF"
+        },
+        {
+            "name": "Tanzania United Republic Of",
+            "iso3": "TZA"
+        }
+    ]
+}
+```
+
+## Pagination
```

Comment:
i'd move this to the top, before the countries and regions endpoints, so it's harder to miss